src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -78,7 +78,6 @@ internal class TradeInputCalculator(
         val subaccount = if (marginMode == MarginMode.Cross) {
             crossMarginSubaccount
         } else {
-            // TODO: incorrect for isolated trades; fix CT-1092
```

Comment:
not actually incorrect, sorry poorly placed TODO which was just to remind me to properly calculate maxLeverage for isolated which we no longer need